jupyter_kernel_mgmt/tests/test_kernelapp.py

```diff
@@ -6,27 +6,28 @@
 import sys
 from tempfile import mkdtemp
 import time
+import json
 
 
 def _launch(extra_env):
     env = os.environ.copy()
     env.update(extra_env)
     return Popen([sys.executable, '-c',
-                  'from jupyter_client.kernelapp import main; main()'],
+                  'from jupyter_kernel_mgmt.kernelapp import main; main()'],
                  env=env, stderr=PIPE)
 
-WAIT_TIME = 10
+
+WAIT_TIME = 20
 POLL_FREQ = 10
 
 
 def test_kernelapp_lifecycle(setup_env):
     # Check that 'jupyter kernel' starts and terminates OK.
-    runtime_dir = mkdtemp()
+    runtime_dir = os.getenv('JUPYTER_RUNTIME_DIR')  # already temp via setup_env fixture
     startup_dir = mkdtemp()
     started = os.path.join(startup_dir, 'started')
     try:
-        p = _launch({'JUPYTER_RUNTIME_DIR': runtime_dir,
-                     'JUPYTER_CLIENT_TEST_RECORD_STARTUP_PRIVATE': started,
+        p = _launch({'JUPYTER_CLIENT_TEST_RECORD_STARTUP_PRIVATE': started,
```

Comment:
nit: We should probably change the variable name so that mixed envs dont' get confused results as this name conflicts with the one in use by the jupyter_client tests.   I was thinking either `JKM_TEST_RECORD_STARTUP_PRIVATE` or `JUPYTER_TEST_RECORD_STARTUP_PRIVATE`.

Reply:
Good point. I'll use the JKM variant.